extensions/Worker.Extensions.CosmosDB/src/Config/CosmosDBBindingOptions.cs

```diff
@@ -23,7 +23,7 @@ internal class CosmosDBBindingOptions
 
         internal ConcurrentDictionary<string, CosmosClient> ClientCache { get; } = new ConcurrentDictionary<string, CosmosClient>();
 
-        internal virtual CosmosClient GetClient(string preferredLocations = "")
+        internal virtual CosmosClient GetClient(CosmosSerializer serializer, string preferredLocations = "")
```

Comment:
If optional, let's either have a default value or an additional overload that doesn't require the caller to pass `null` here.